packages/context/src/__tests__/unit/context.unit.ts

```diff
@@ -533,6 +533,21 @@ describe('Context', () => {
     });
   });
 
+  describe('getOwnerContext', () => {
```

Comment:
Ditto: Let's land this improvement in a standalone pull request please. I'd like to see a dedicated entry in the CHANGELOG.

Reply:
+1 (ignore if feedback applied).